cartridges/sfra_demo/cartridges/sfra_demo/cartridge/controllers/AppendExample2.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var server = require('server');
+var x = require('*/cartridge/controllers/AppendExample');
+
+server.extend(x);
```

Comment:
'x' - is it a variable name that was in the provided cartridge? Please, avoid using meaningless variable names

Reply:
Variable renamed.
It was named by author of this cartridge.  I didn't see it.